src/main/java/io/qdrant/client/VectorInputFactory.java

```diff
@@ -107,10 +110,40 @@ public static VectorInput vectorInput(UUID id) {
   /**
    * Creates a {@link VectorInput} from a {@link PointId}
    *
-   * @param id The pint id
+   * @param id The point id
    * @return a new instance of {@link VectorInput}
    */
   public static VectorInput vectorInput(PointId id) {
     return VectorInput.newBuilder().setId(id).build();
   }
+
+  /**
+   * Creates a {@link VectorInput} from a {@link Document}
+   *
+   * @param document An instance of {@link Document}
+   * @return a new instance of {@link VectorInput}
+   */
+  public static VectorInput vectorInput(Document document) {
+    return VectorInput.newBuilder().setDocument(document).build();
+  }
+
+  /**
+   * Creates a {@link VectorInput} from a an {@link Image}
+   *
+   * @param image An instance of {@link Image}
+   * @return a new instance of {@link VectorInput}
+   */
+  public static VectorInput vectorInput(Image image) {
+    return VectorInput.newBuilder().setImage(image).build();
+  }
+
+  /**
+   * Creates a {@link VectorInput} from a {@link InferenceObject}
+   *
+   * @param object An instance of {@link InferenceObject}
+   * @return a new instance of {@link VectorInput}
+   */
+  public static VectorInput vectorInpu(InferenceObject object) {
+    return VectorInput.newBuilder().setObject(object).build();
+  }
```

Comment:
Let's comment these out until we release inference.

Reply:
I have the same question as https://github.com/qdrant/qdrant-dotnet/pull/77#discussion_r1917022011

Reply:
That's fine for now 👍. It makes them a bit harder to discover.

Reply:
Done.